crowbar_framework/app/models/backup.rb

```diff
@@ -172,6 +176,8 @@ def save_archive
     self.version = meta["version"]
     self.size = path.size
     self.created_at = Time.zone.parse(meta["created_at"])
+    # 20151222144602_create_backups.rb
```

Comment:
@vuntz to have some comment, can you help me here a bit to find a reasonable explanation for this in your words?